tailwind.config.js

```diff
@@ -11,14 +20,20 @@ module.exports = {
                 blue: {
                     lighter: '#ECF5FF',
                     light: '#086BCF',
+                    lightTransparent: '#338AFF33',
```

Comment:
escolha uma das duas formas de aplicar o token, não cheguei a olhar como estava no Figma.

```suggestion
                    light-100: '#338AFF33',
                    ight-transparent: '#338AFF33',
```